SYCL/ESIMD/api/functional/ctors/ctor_copy.cpp

```diff
@@ -11,16 +11,17 @@
 // The current "REQUIRES" should be replaced with "gpu" only as mentioned in
 // "XREQUIRES".
 // UNSUPPORTED: cuda, hip
-// XRUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
-// XRUN: %GPU_RUN_PLACEHOLDER %t.out
-// RUN: false
-// XFAIL: *
-// TODO Unexpected static_assert was retrieved while calling simd::copy_from()
-// function. The issue was created (https://github.com/intel/llvm/issues/5112)
-// and the test must be enabled when it is resolved.
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
 //
 // Test for esimd copy constructor.
 
+// The following issues for simd<T,32> observed:
+// - freeze with T in {char, unsigned char, singned char};
+// - runtime failure with T in {short, unsigned short}.
```

Comment:
please specify the link to the inner source issue as the reason of disabling a part of verifications: https://github.com/intel/llvm/issues/5245
for example:
```suggestion
// The following issues for simd<T,32> observed:
// - freeze with T in {char, unsigned char, singned char};
// - simd<T, 32> vector filled with unexpected values from 16th element with T
// in {short, unsigned short}. The issue was created
// https://github.com/intel/llvm/issues/5245
```

Reply:
Great hint, thanks. Applied by 4d8c8661e0c4170fb1ca9304c6e38c9b800bfd39 by more laconic way, to avoid duplication with the issue itself. Let me know if it's too short from your perspective.